app/controllers/review_controller.rb

```diff
@@ -36,6 +36,16 @@ def edit
     end
   end
 
+  def discard
+    rugged_delete_branch(params[:id])
+    begin
```

Comment:
Maybe if there is a begin we could have a rescue? Just in case it throws an error.

Reply:
heh. unfortunately, if the term does not exist, the response IS an error.

Reply:
irb(main):003:0> Term.exists? "bleha/foo"
[INFO] fetch_cached_graph(http://opaquenamespace.org/ns/bleha/foo) not found in triplestore cache (http://blazegraph:9999/blazegraph/namespace/development/sparql)
[ERROR] TriplestoreRepository.statements failed with TriplestoreException: graph not found in cache, from_remote:, skipping fetch from remote graph
=> false

Reply:
flash[:notice] = Term.exists? params[:id] ? "Modifications to #{params[:id]} are discarded." : "#{params[:id]} has been discarded."